packages/admin-stories/src/docs/components/RouterTabs/stories/RouterTabsForceRender.stories.tsx

```diff
@@ -0,0 +1,180 @@
+                                        >
+                                            <SaveButton
+                                                color={"primary"}
+                                                variant={"contained"}
+                                                saving={submitting}
+                                                hasErrors={hasSubmitErrors}
+                                                type="button"
+                                                onClick={async () => {
+                                                    handleSubmit();
+                                                }}
+                                            >
+                                                Save
+                                            </SaveButton>
+                                            <SaveButton
+                                                color={"primary"}
+                                                variant={"contained"}
+                                                saving={submitting}
+                                                hasErrors={hasSubmitErrors}
+                                                onClick={async () => {
+                                                    handleSubmit();
+                                                }}
+                                            >
+                                                Save and go back
+                                            </SaveButton>
+                                        </SplitButton>
+                                    </ToolbarActions>
+                                </Toolbar>
+                                <RouterTabs>
+                                    <RouterTab label="Form 1" path="" forceRender>
+                                        <Card variant="outlined">
+                                            <CardContent>
+                                                <Field label="Foo" name="foo" component={FinalFormInput} />
+                                            </CardContent>
+                                        </Card>
+                                    </RouterTab>
+                                    <RouterTab label="Form 2" path="/form2" forceRender>
+                                        <Card variant="outlined">
+                                            <CardContent>
+                                                <Field label="Bar" name="bar" component={FinalFormInput} />
+                                            </CardContent>
+                                        </Card>
+                                    </RouterTab>
+                                </RouterTabs>
+                            </>
+                        );
+                    }}
+                </FinalForm>
+            </div>
+        );
+    })
+    .add("RouterTabs without forceRender", () => {
```

Comment:
Shouldn't this be the first story, as it's the basic example?

Reply:
I don't think so because the stories in this file focus on the forceRender prop (`RouterTabsForceRender.stories.tsx`). 

The `RouterTabs without forceRender` example doesn't work properly. I only added it to show in which cases you should use `forceRender` (and how the form behavior changes). I plan to use these stories for the Docs (https://github.com/vivid-planet/comet-admin/pull/620#discussion_r884603905)